nala/features/masccd/masccd.spec.js

```diff
@@ -37,6 +37,11 @@ module.exports = {
         linkText: 'See terms',
         linkUrl: '',
         iconUrl: '',
+        cardDaaLH: 'ccsn',
+        linkAnalyticsId: 'see-terms',
+        linkDaaLL: 'see-terms-2',
+        ctaAnalyticsId: 'buy-now',
+        ctaDaaLL: 'buy-now-2',
```

Comment:
@3ch023 this value here should be "buy-now-1". The tests fails

Reply:
thx,done